lib/kvao/ttl.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var assert = require('assert');
+var utils = require('../utils');
+
+/**
+ * Get remaining expiration (TTL) for a given key.
+ *
+ * @param {String} key
+ * @param {Object} options
+ * @callback cb
+ * @param {Error} error
```

Comment:
The callback takes the second argument. Please document it and explain what value is returned when no TTL was set.